src/main/webapp/itempagescript.js

```diff
@@ -0,0 +1,23 @@
+function getItemId() {
+  const queryString = window.location.search;
+  console.log(queryString);
+}
+
+async function getItemPageComments() {
+  const response = await fetch("/itempagedata");
+  const comments = await response.json();
+  const commentContainer = document.getElementById("comment-container");
+  comments.forEach((commentObject) => {
+    const date = new Date(commentObject.timestamp);
+    commentContainer.appendChild(
+      createListElement(
+        commentObject.message + " - " + "(" + date.toUTCString() + ")"
+      )
+    );
+  });
+}
```

Comment:
Nit: leave empty line between the end of a function and the beginning of another one.